index.html

```diff
@@ -163,6 +163,32 @@ <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3>
                   </li>
               </ul>
              </section>
+
+             <section class="form">
+                <form action="https://formspree.io/f/mnqygqol" method="post">
+                <h2 class="form-header">
+                    Contact me
+                </h2>
+                <p>
+                    If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!
+                </p>
+                <ul class="ul_form">
+                    <li>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly make sure Flexbox is used to layout the elements of your contact section and the form itself so as to make the section responsive and also gives you the added advantage of laying out the elements the way you want to improve user experience 👍🏽 .